Userland/Libraries/LibJS/Lexer.cpp

```diff
@@ -836,7 +836,7 @@ Token Lexer::next()
 
     if (m_hit_invalid_unicode.has_value()) {
         value_start = m_hit_invalid_unicode.value() - 1;
-        m_current_token = Token(TokenType::Invalid, "Invalid unicode codepoint in source",
+        m_current_token = Token(TokenType::Invalid, String::from_deprecated_string("Invalid unicode codepoint in source").release_value_but_fixme_should_propagate_errors(),
```

Comment:
Currently the three changes (this one and the two below) are about Token.
I think we should either drop these changes or update Token in a seperate commit.

Reply:
This one should actually be:
```c++
String::from_utf8("Invalid unicode codepoint in source"sv).release_value_but_fixme_should_propagate_errors()
```

Reply:
Fixed.